pkg/shell/manifest.json

```diff
@@ -26,6 +26,12 @@
         "zh-cn": "中文（中国）",
         "zh-tw": "中文（台灣）"
     },
+    "rtl_locales": [
```

Comment:
This could be generated as are `locales` but I don't think it is necessary. Right now this list contains all rtl languages from languages_map - those are all that Weblate supports.